test/cpp/qps/client.h

```diff
@@ -429,12 +429,57 @@ class ClientImpl : public Client {
           config.server_targets(i % config.server_targets_size()), config,
           create_stub_, i);
     }
-    std::vector<std::unique_ptr<std::thread>> connecting_threads;
+    int connect_deadline_seconds = 10;
+    /* Allow optionally overriding connect_deadline in order
+     * to deal with benchmark environments in which the server
+     * can take a long time to become ready. */
+    char* channel_connect_timeout_str =
```

Comment:
readability: would it make sense to extract the logic of connecting multiple channels to a separate method?

Reply:
good point, I think so.